doc/accessibility/index.md

```diff
@@ -0,0 +1,118 @@
+# Accessibility
+
+!!! note 
+Work in progress, request for discussion
+
+
+**Status**: first sketch, work in progress, request for collaboration
+
+**Date**: Updated 2025-04-03
+
+**Governance**: To Be Discovered; potentially a combo of this repo participants, DHCW UI/UX, NHS Wales UCB peers, etc.
+
+## Context
+
+We want to ensure accessibility for users, including users with disabilities. This is a fundamental aspect of providing an inclusive user experience, and an ethical principle, and a legal requirement.
+
+The team has to decide on how to integrate accessibility into the product’s design and architecture to ensure compliance with accessibility standards and deliver an experience that is usable for all users.
```

Comment:
I'd emphasise that compliance with standards is the minimum and we seek to validate through testing with real users (I know this is covered later, but is important as I have seen teams claim to have accessible software as it passes at technical test but fails miserably when a real user utilising assistive technologies tries to use it.

Reply:
Great point. Done.